<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookBeats</class>
 <widget class="QMainWindow" name="BookBeats">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>687</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>7</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 170, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="navbar" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>771</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:1 rgba(0, 0, 0, 108));
color: rgb(255, 255, 255);</string>
    </property>
    <widget class="QLabel" name="logo">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>131</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>img/logo.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="appName">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>10</y>
       <width>221</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt; BookBeats&lt;sup&gt;(BB)&lt;/sup&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="linkedin_btn">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>50</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
       <italic>true</italic>
       <underline>true</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent;
border:0;</string>
     </property>
     <property name="text">
      <string>Linkedin</string>
     </property>
    </widget>
    <widget class="QPushButton" name="github_btn">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>50</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
       <italic>true</italic>
       <underline>true</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent;
border:0;</string>
     </property>
     <property name="text">
      <string>Github</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="content" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>701</width>
      <height>381</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:1 rgba(0, 0, 0, 108));</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>541</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
border-radius5px;</string>
     </property>
     <property name="text">
      <string>PDF into Audio MP3 File</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="openfile">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>181</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 127);
border-radius:5px;</string>
     </property>
     <property name="text">
      <string>Open File</string>
     </property>
    </widget>
    <widget class="QComboBox" name="LanguageList">
     <property name="geometry">
      <rect>
       <x>389</x>
       <y>120</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent;
color: rgb(255, 255, 255);
border:1px solid black;

padding:5px;
padding-left:15px;</string>
     </property>
     <property name="currentIndex">
      <number>11</number>
     </property>
     <property name="iconSize">
      <size>
       <width>21</width>
       <height>21</height>
      </size>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
     <item>
      <property name="text">
       <string>Afrikaans</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Arabic</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Bengali</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Bosnian</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Bulgarian</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Catalan</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Chinese </string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Croatian</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Czech</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Danish</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Dutch</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>English (Australia)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>English (India)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>English (UK)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>English (US)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Filipino</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Finnish</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>French</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>German</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Greek</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Gujarati</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Hebrew</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Hindi</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Hungarian</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Icelandic</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Indonesian</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Italian</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Japanese</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Javanese</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Kannada</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Khmer</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Korean</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Latvian</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Lithuanian</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Malayalam</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Marathi</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Nepali</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Portuguese (Brazil)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Portuguese (Portugal)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Punjabi</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Romanian</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Russian</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Serbian</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sinhala</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Slovenian</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Spanish</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sundanese</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Swahili</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Swedish</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Tamil</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Telugu</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Thai</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Turkish</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ukrainian</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Urdu</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Vietnamese</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Welsh</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Xhosa</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Zulu</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="select_lang">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>120</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent;
color: rgb(255, 255, 255);
border:0;</string>
     </property>
     <property name="text">
      <string>Select Language :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lfileName">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>281</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background:0;</string>
     </property>
     <property name="text">
      <string>Enter Name For Audio File *.MP3</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="audiofilename">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>190</y>
       <width>331</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
padding:3px;
padding:15px;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Download">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>300</y>
       <width>311</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(77, 231, 113);
border-radius:5px;</string>
     </property>
     <property name="text">
      <string>Download </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>310</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <underline>false</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);

background:0;</string>
     </property>
     <property name="text">
      <string>After Downloaded it will Start Playing</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>520</y>
      <width>381</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1.Open Pdf File .</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>550</y>
      <width>381</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>2.Select Language.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>580</y>
      <width>401</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>3. Enter Audio file Name and  Click  Download Button.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>600</y>
      <width>781</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:red;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Please Wait for minute for Processing form every word to audio it will take few minutes.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
